client.js

```diff
@@ -217,21 +223,53 @@ async function selectCountriesAndCities(hzClient) {
     console.log("--------------------");
 }
 
+//
+//   This example shows how to work with Hazelcast maps, where the map is
+//   updated continuously.
+//
+//   @param client - a {@link HazelcastInstance} client.
```

Comment:
HazelcastInstance does not exist in Node.js client 

```suggestion
//   @param client - a {@link HazelcastClient} client.
```